src/gluonts/model/rotbaum/_predictor.py

```diff
@@ -168,6 +173,11 @@ def __init__(
         self.quantiles = quantiles
         self.model_list = None
 
+        logger.info(
+            "If using the Evaluator class with a TreePredictor, set num_workers=0. The TreePredictor and Evaluator "
```

Comment:
Can you shorten the lines, so that they ideally don't exceed 79 characters each?